doc/sphinx/using/libraries/stdlib-internals.rst

```diff
@@ -0,0 +1,139 @@
+Stdlib internals
+================
+
+These commands and tactics are now provided in the Stdlib
+hence documented in its refman.
```

Comment:
```suggestion
These commands and tactics are provided by the standard library, and hence documented in its reference manual.
```
Also, I would just remove all the internal tactics and only keep the commands and tactics that a user might actually have a chance to use, that is, `Function`, `functional foo`, `Functional Scheme`, `Add Ring`, `Add Field`, `Add Zify`.

Reply:
Unfortunately, I have no idea how to do that with the docgram tool.

Reply:
Are you asking how to drop these commands from the Coq manual?  If the underlying code remains in Coq, you would have to delete them in `common.mlg_edit`.  The tool will otherwise complain about undocumented commands and tactics, as it is designed to do.

On the stdlib side, the command and tactic descriptions currently refer to a number of grammar nonterminals that are currently hyperlinked.  Stdlib won't be able to include those hyperlinks if it has its own reference manual.  The missing nonterminals should still be in a distinguished font (but without the hyperlink) so they're not confused with literals that appear in tactics and commands (e.g. "in", "with").  So the rendering code would have to be changed somewhat.  And you'd need to drop all the tactics and commands that are in the Coq manual or disable the checks for missing commands.

It would be better to try generating the stdlib documentation before changing the Coq side since there will be more to deal to generate it in stdlib.  That would let people review the resulting manual and decide if splitting the doc really makes sense.  If the code implementing commands/tactics is still in Coq, the doc should stay there.